apps/emqx/src/emqx_broker.erl

```diff
@@ -168,9 +167,7 @@ with_subid(SubId, SubOpts) ->
     maps:put(subid, SubId, SubOpts).
 
 do_subscribe(Topic, SubPid, SubOpts) when is_binary(Topic) ->
-    %% FIXME: subscribe shard bug
-    %% https://emqx.atlassian.net/browse/EMQX-10214
```

Comment:
was it fixed in this PR?

Reply:
I haven't completely understood the issue outlined in the ticket, but as far as I could tell it no longer applies.

> there were 5 {shard, _} tuple in subscribers list.

Specifically, this no longer the case.